api/src/main/java/org/openmrs/module/openhmis/cashier/ModuleSettings.java

```diff
@@ -28,6 +28,7 @@
 	public static final String ROUNDING_DEPT_ID = "openhmis.cashier.roundingDeptId";
 	public static final String SYSTEM_RECEIPT_NUMBER_GENERATOR = "openhmis.cashier.systemReceiptNumberGenerator";
 	public static final String ADJUSTMENT_REASEON_FIELD = "openhmis.cashier.adjustmentReasonField";
+    public static final String BILL_ADJUSTMENT_SETTING ="openhmis.cashier.billadjustmentSetting";

```

Comment:
This setting name is a bit generic. If this is the setting to enable or disable the ability to adjust bills then the name should be something like allowBillAdjustments.